tests/mbed/host/host.c

```diff
@@ -40,11 +43,11 @@ void datafileloc(char* data_file_name, char* path)
 {
     char* tail = "../enc/";
 #ifdef PROJECT_DIR
-    strcpy(path, PROJECT_DIR);
+    strcpy_s(path, PATH_LEN, PROJECT_DIR);
```

Comment:
caller of datafileloc should pass in size of `path` buffer

Reply:
Great, this is exactly what I need! I just updated and will mark this as resolved.